INSTALL.md

```diff
@@ -513,5 +496,97 @@ $ git submodule update
 
 Then continue with “Crosscompiling from a released tarball” above.
 
+VII. Compiling on Windows with msys2
+-----------------------------------------
+
+TSC can be compiled on Windows using msys2. The msys2 suite provides a convenient Bash shell for Windows, along with an expanding
+repository of prebuilt packages.
+
+
+### Installing and updating msys2 ###
```

Comment:
Minor nitpick: All other headings on this level in the file use numbering, so you should add arabic numbering here as well.

Reply:
Errghs. Somehow I'm not compatible with GitHub's commit comment interface. I have no idea why it now shows "Pending".... Please ignore my ignorance

Reply:
Fixed. Also, it looks like I messed up the contents. That may actually be because I used an external markdown editor...